src/languageservice/services/yamlSchemaService.ts

```diff
@@ -14,7 +14,7 @@ import * as nls from 'vscode-nls';
 import { convertSimple2RegExpPattern } from '../utils/strings';
 const localize = nls.loadMessageBundle();
 
-export declare type CustomSchemaProvider = (uri: string) => Thenable<string>;
+export declare type CustomSchemaProvider = (uri: string) => Thenable<string[]>;
```

Comment:
Do we need to consider compatibility with the current clients? How about making this return string or string array to keep the compatibility??
```
export declare type CustomSchemaProvider = (uri: string) => Thenable<string | string[]>;
```